From 6703935294e0f3fe39b41490d3eb8665bcc9f013 Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <Guoqing.Jiang@windriver.com>
Date: Tue, 16 Jul 2013 17:03:14 +0800
Subject: [PATCH 26/57] ARM: keystone: add TI_KEYSTONE flag

Commit 7a90a35589e527d8f33c2a22bad6a2306aeb9513 from arago tree
git://arago-project.org/git/projects/linux-keystone.git releases/03.00.00.11/master

Signed-off-by: Sandeep Paulraj <s-paulraj@ti.com>
Signed-off-by: Guoqing Jiang <Guoqing.Jiang@windriver.com>
---
 arch/arm/Kconfig               | 1 +
 arch/arm/mach-keystone/Kconfig | 3 +++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index f06daf6..efa455a 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -379,6 +379,7 @@ config ARCH_KEYSTONE
 	select HAVE_SCHED_CLOCK
 	select HAVE_SMP
 	select ZONE_DMA if ARM_LPAE
+	select TI_KEYSTONE
 	help
 	  Support for boards based on the Texas Instruments Keystone family of
 	  SoCs.
diff --git a/arch/arm/mach-keystone/Kconfig b/arch/arm/mach-keystone/Kconfig
index a00b97da..2409b93 100644
--- a/arch/arm/mach-keystone/Kconfig
+++ b/arch/arm/mach-keystone/Kconfig
@@ -1,5 +1,8 @@
 if ARCH_KEYSTONE
 
+config TI_KEYSTONE
+	bool
+
 menu "TI KeyStone Implementations"
 
 endmenu
-- 
1.8.4.93.g57e4c17

